### ISSUE TITLE
Error making vtypes when making profile on OSX El Capitan 10.11

### ISSUE THREAD

**Issue by taylormm** [volatilityfoundation/volatility#246]
python tools/mac/convert.py ../Kernels/converted-10.11.64bit.dwarfdump > ../Kernels/10.11.64bit.vtypes

Traceback (most recent call last):
  File "tools/mac/convert.py", line 561, in <module>
    main()
  File "tools/mac/convert.py", line 558, in main
    parse_dwarf()     
  File "tools/mac/convert.py", line 398, in parse_dwarf
    parser.feed_line(line)
  File "tools/mac/convert.py", line 129, in feed_line
    self.process_statement(**parsed)
  File "tools/mac/convert.py", line 219, in process_statement
    self.id_to_name[statement_id] = [self.base_type_name(data)]
  File "tools/mac/convert.py", line 101, in base_type_name
    return self.tp2vol[data['AT_name']]
KeyError: 'sizetype'

My dwarfdump was created by the kernel.dSYM from http://adcdownload.apple.com/Developer_Tools/Kernel_Debug_Kit_10.11_build_15A284/Kernel_Debug_Kit_10.11_build_15A284.dmg

**Comment by atcuno** [comment]
I created a profile for it:

https://github.com/volatilityfoundation/profiles/tree/master/Mac/10.11

Please let me know if you have any issues.

**Comment by gleeda** [comment]
Just hit this error with 10.11.1_Build_15B42.  it would be good to update the script to handle this

**Comment by njonjo1985** [comment]
Also just hit this error with 10.11.1_Build_15B42. @atcuno How did you go about creating the profile for 10.11.0 ? Could you please share with us so that we could do the same for 10.11.1?

**Comment by williamshowalter** [comment]
After looking at the how the kernel debug kit output has changed, I updated the convert.py script to be compatible with newer versions. I think @atcuno had done something similar with the ones he was publishing in the profiles repo himself, but it seemed like there were a few people looking to make their own. Pull request #296 should fix this - I've tested on 10.11.3 & 10.11.4 successfully, and verified it still works to generate a profile to use with the 10.9.3 images provided in AoMF.
